src/main/constraints/90_quick_launches.sql

```diff
@@ -0,0 +1,24 @@
+SET search_path = public, pg_catalog;
+
+ALTER TABLE ONLY quick_launches
+    ADD CONSTRAINT quick_launches_creator_fkey
+    FOREIGN KEY (creator)
+    REFERENCES users(id);
+
+ALTER TABLE ONLY quick_launches
+    ADD CONSTRAINT quick_launches_submission_id_fkey
+    FOREIGN KEY (submission_id)
+    REFERENCES submissions(id);
+
+ALTER TABLE ONLY quick_launches
+    ADD CONSTRAINT quick_launches_app_id_fkey
+    FOREIGN KEY (app_id)
+    REFERENCES apps(id);
```

Comment:
The foreign key for the `app_id` column will have to be removed if we need to support quick launches for Agave apps.